src/main/app/Event/StrictDispatcher.php

```diff
@@ -69,6 +92,15 @@ public function dispatch($eventName, $shortEventClassName, array $eventArgs = []
             throw new MandatoryEventException("No listener is attached to the '{$eventName}' event");
         }
 
+        $eventSubscriber = new SecurityEventSubscriber(
+            $this->em,
+            $this->client,
+            $this->security,
+            $this->requestStack,
+            $this->translator
+        );
```

Comment:
Event listeners/subscribers must be registered as services tagged with `kernel.event_subsciber`, not instantiated manually. Ping me if you have an issue with that

Reply:
Yes, the file/service is there but I forgot to remove this piece of code, it's done thank you :)